test/test_util_get_base_title.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2021 Laurent Monin
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from test.picardtestcase import PicardTestCase
+
+from picard.const import DEFAULT_COPY_TEXT
+from picard.util import get_base_title
+
+
+class GetBaseTitle(PicardTestCase):
+    def test_base_title_0(self):
+        test_title = 'title'
+        title = get_base_title(test_title)
+        self.assertEqual(title, 'title')
+
+    def test_base_title_1(self):
+        test_title = 'title {0}'.format(_(DEFAULT_COPY_TEXT),)
```

Comment:
That's a bad idea to rely on translated texts in tests, instead I would pass the string as an argument to `get_base_title()`

Reply:
See my comment above.

It would be more readable without `format()`:

```python
test_title = 'title copy (1)'
title = get_base_title(test_title, 'copy')
```
	
@zas